instant-gram/ViewControllers/LoginViewController.m

```diff
@@ -44,6 +44,7 @@ - (void)registerUser {
             [self showAlert:error.localizedDescription];
         } else {
             NSLog(@"User registered successfully");
+            [self performSegueWithIdentifier:@"loginSegue" sender:nil];
```

Comment:
I think this whole if-statement could be in a method that you call from here and from `loginUser`